### PR TITLE
Additional special notes (suggestion)

### PR DESCRIPTION
**Pull request by walodar**
Hello, I have a suggestion for additional special notes for a number of units, similar to the recently added ones (for example for the Spearman advancements).
Thanks for considering this!

### CODE REVIEW

**Comment by nemaara** [comment]
I can see this being useful for certain cases, but this seems super unnecessary for every unit. We don't need to spell out that the Blood Bat has one more movepoint than the Vampire bat, for instance.

By special cases, I mean something like the Ranger/Avenger, which has a significantly buffed movetype that isn't necessarily that easy to tell at a glance from the table (since it's so long). Things like resistance increases are very easy to see at a glance, so I think that's not needed for a special note (there are some exceptions here, like for the Silver Mage where it's a big unit feature rather than just a small 10% incremental increase).

File: data/core/units/dunefolk/Raider.cfg [hunk 1]
```diff
@@ -18,6 +18,9 @@ units/dunefolk/rider/#enddef
     cost=34
     usage=archer
     description= _ "Though rarely found in organized armies, Raiders are a staple among the nomadic Dunefolk, who regularly ambush rival caravans and camps at night. In these scenarios, raw power is of little concern. The greatest importance is placed on speed—striking quickly, spreading as much chaos with their torches in the shortest time possible, allows these riders to get in and out of the blitz without any fear of counterattack. Raiders are the fastest of the Dunefolk, capable of outpacing nearly anything they might encounter in the sandy deserts."
+    [special_note]
+        note= _ "Unlike the liminal Dune Riders, Dune Raiders are chaotic. They also have an additional movepoint, a higher resistance to ‘blade’ attacks and worse defense in mountains."
```
Comment (sevu) [inline data/core/units/dunefolk/Raider.cfg]: That's twice Dune Riders here

**Comment by walodar** [comment]
Hello, you are of course right that the notes are not necessary, they give no information that cannot be found by looking at the numbers.

Currently, a few simple special notes are already there, for example for the 10% fire resistance of the Red Mage. My motivation was to offer the same pointers everywhere for the sake of unification and convenience for players.

But this is only a suggestion, I am happy if this is deemed useful, but of course it's also okay if only a few ones or none of them are added.

Thanks!

**Comment by CelticMinstrel** [comment]
I feel like about 90% of these are excessive and not all that useful. I think the main ones that stood out as _maybe_ useful are the dunefolk alignment changes and the notes for elvish fliers. Actually, I'm still mildly leaning towards adding a general special note for _all_ liminal units (I think some people disagree though which is why I haven't done it).

**Comment by nemaara** [comment]
I don't think dunefolk alignment changes are necessary to spell out in special notes. It's clear enough when the alignment itself is listed on its own. With elvish fliers, I feel like comparing them to regular fliers is excessive (although it does make me wonder why they wouldn't be able to fly over unwalkable terrain, but that's another matter). Come to think of it, I don't really like the special note for the Ranger/Avenger line, it doesn't seem necessary to me (but that's something for a different PR). Basically, I agree with celmin in that these are excessive. Closing this PR.

**Comment by CelticMinstrel** [comment]
Strangely, the ranger/avenger special note was also one of the few that seemed okay to me (but it was already there; the revision to it did seem excessive).

**Pull request closed without merging**